src/collections/resources/less-than-one-shot-learning-learning-n-classes-from-mlessn-samples-pdf-article.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "\"Less Than One\"-Shot Learning: Learning N Classes From M<N Samples"
+title: "\"Less Than One\"-Shot Learning: Learning N Classes From M&lt;N Samples"
```

Comment:
You are right. Having a `<` in any text fields in a resource will cause the build to fail. We probably need to fix this before the release.